src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -211,10 +207,10 @@ private JsonObject getPermissionsObject() {
      * @return List of shards
      */
     public List<Shard> getShards() {
-        HttpResponse response = null;
-        HttpGet get = new HttpGet(buildUri(db.getDBUri()).path("/_shards").build());
+        InputStream response = null;
+        //HttpGet get = new HttpGet(buildUri(db.getDBUri()).path("/_shards").build());
         try {
-            response = client.executeRequest(get);
+            response = client.get(buildUri(db.getDBUri()).path("/_shards").build());
```

Comment:
Does this `/` before `_shards` need removing? In other places they seem to have been.